C4SGSvc/src/main/java/org/c4sg/constant/C4sgConstants.java

```diff
@@ -0,0 +1,11 @@
+package org.c4sg.constant;
+
+public class C4sgConstants {
```

Comment:
That's fine, one size fits all for now. We can split up to several constants when application grows.